client/js/editmode.js

```diff
@@ -1,9 +1,9 @@
 //This section holds the functions that generate the JSON of the widgets in the add widget overlay
 function generateCardDeckWidgets(id, x, y, addCards) {
   const widgets = [
-    { type:'holder', id, x, y, dropTarget: { type: 'card' } },
+    { type: 'holder', id, x, y, dropTarget: { type: 'card' } },
     {
-      id: id+'B',
+      id: id + 'B',
```

Comment:
This whole file contains hundreds of whitespace changes. Feels very unnecessary and this causes conflicts with other open PRs for no reason (already happened with the canvas line width PR).